flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamOverAggregate.scala

```diff
@@ -112,7 +113,12 @@ class DataStreamOverAggregate(
               "condition.")
         }
       case _: RowTimeType =>
-        throw new TableException("OVER Window of the EventTime type is not currently supported.")
+        if (overWindow.lowerBound.isUnbounded && overWindow.upperBound.isCurrentRow) {
+          createUnboundedAndCurrentRowEventTimeOverWindow(inputDS)
```

Comment:
Hi @fhueske I had created a new JIRA. [FLINK-6200](https://issues.apache.org/jira/browse/FLINK-6200) for the `RANGE` case. But I do not have permission to modify this JIRA's name. So, can you help me to do it? :)
Hi, @hongyuhong  I'm glad to hear that you want to continue do the `RANGE` case. Feel free you want to take [FLINK-6200](https://issues.apache.org/jira/browse/FLINK-6200).